lib/ImportStatement.js

```diff
@@ -149,10 +149,7 @@ export default class ImportStatement {
    * @return {Array} an array that can be used in `sort` and `uniq`
    */
   toNormalized(): Array<string> {
-    if (!this.defaultImport && !this.hasNamedImports() && this.hasSideEffects) {
-      return [this.path];
-    }
-    return [this.defaultImport || '', ...this.localNames()];
+    return [this.defaultImport !== 'React', this.path];
```

Comment:
Why do you want `React` to be on top?

Reply:
I'm not sure I understand. I haven't come across any project where React has to be at the top.

Reply:
I guess I was wrong - it is only standard in the firm I work in. Therefore I left it to [this.path] only.